generic/sysbench.py

```diff
@@ -48,9 +47,26 @@ def run_cmd(self, cmdline):
             self.fail("The sysbench failed: %s" % details)
 
     def setUp(self):
-        softmanager = SoftwareManager()
-        if not softmanager.check_installed('sysbench') and not softmanager.install('sysbench'):
-            self.cancel("sysbench package is required to continue the test")
+        if not process.system("which sysbench"):
+            softmanager = SoftwareManager()
+            for package in ("autoconf", "libtool", "make"):
+                if not softmanager.check_installed(package) and not softmanager.install(package):
+                    self.error(
+                        "Fail to install %s required for this test." % package)
+            if not softmanager.check_installed('sysbench') and not softmanager.install('sysbench'):
```

Comment:
i think should be before other package installation if it failed than package installed is required to build upstream sysbench

Reply:
can be part of else block

Reply:
done with the required changes.